xml/System.Globalization/CultureInfo.xml

```diff
@@ -2064,7 +2066,6 @@ csc /resource:GreetingStrings.resources Example1.cs
 |Vietnamese|vi|en-US|  
 |Vietnamese (Vietnam)|vi-VN|en-US|  
   
- Your application can use <xref:System.Globalization.CultureAndRegionInfoBuilder> to create a replacement for a culture, and that culture can have a console fallback culture of your choosing.  
```

Comment:
Removed this sentence well because Locale Builder tool seem cannot specify for a console fallback culture.